app/commands.js

```diff
@@ -74,4 +74,13 @@ export default (app) => {
     return `${giphyUrl}search?api_key=${giphyToken}&q=${encodeURIComponent(queryString)}&limit=${limit}`;
   }
 
+  app.command('jaja', (ctx) => {
+    ctx.reply(staticResponses['jaja']);
+  });
+  
+  app.command('chupala', (ctx) => {
+    const index = Math.floor((Math.random() * staticResponses.chupala.length-1) + 1);
```

Comment:
Random number could be generated from `getRandomNum` function if it receives a limit